src/app/containers/Timestamp/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { number } from 'prop-types';
+import Timestamp from '../../components/Timestamp';
+
+const formatTimestamp = timestamp => {
+  const dateTime = new Date(timestamp * 1000);
+  const day = dateTime.toLocaleString('en-GB', {
+    day: 'numeric',
```

Comment:
We should add a test or stories to ensure this renders a single digit day as `7` and not as `07`